versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/isolating-namespaces.mdx

```diff
@@ -9,7 +9,7 @@ and protects applications from each other, and reduces blast radius of failure c
 The following figure illustrates how the namespaces are isolated from different namespaces on a worker cluster.
 Namespaces are isolated with respect to sending and receiving data traffic to other namespaces in a cluster.
 
-![alt](/images/Namespace-Isolation-OS.png)
+![alt](/img/Namespace-Isolation-OS.png)
```

Comment:
we have used `images` as a standard folder and I think that would be part of the CSS-defined folder. 

@that-backend-guy  - can you confirm? 

So, we can retain the name of the folder as `images`.

Reply:
@veenamj-avesha , yes, we can move from `images` to `/static/img` directory